src/ConfirmProvider.js

```diff
@@ -66,6 +67,12 @@ const buildOptions = (defaultOptions, options) => {
       DEFAULT_OPTIONS.acknowledgementCheckboxProps),
     ...(options.acknowledgementCheckboxProps || {}),
   };
+  // We must accept 'false' values, if present.
+  const closeOnParentUnmount = typeof options.closeOnParentUnmount !== 'undefined' ?
+    options.closeOnParentUnmount :
+    typeof defaultOptions.closeOnParentUnmount !== 'undefined' ?
+      defaultOptions.closeOnParentUnmount :
+      DEFAULT_OPTIONS.closeOnParentUnmount;
```

Comment:
I don't think we need this, the `...` below already merges options. The logic above is for objects where we want to deep merge the options.

```suggestion
```